transient_model.py

```diff
@@ -0,0 +1,47 @@
+import numpy as np
+from build_matrix_A import build_matrix_A
+from build_matrix_b import build_matrix_b
+from matrix_solver import solve_matrix
+from build_A_analytical import build_A_analytical
+from generate_mesh import generate_mesh

```

Comment:
Might consider putting many/all of these functions in the same file - possibly this file.